### ISSUE TITLE
Passkey now responding to connections with 403

### ISSUE THREAD

**Issue by fordan** [mrozekma/gencon-hotel-check#1]
It looks like something changed on the Passkey side when searching for a hotel without having an existing reservation: it is now returning a HTTP Error 403: Forbidden message. It's not as simple as User Agent filtering; I tried changing the Python-urllib/2.7 User Agent to match Chrome for Window's string and it still fails.

**Comment by vicpylon** [comment]
Cvent/passkey was informed of their poor security practices and made significant changes.  I first noticed the changed a couple of weeks ago when my key changed.  They may have taken additional steps to prevent attacks.  Here is the original bug report that might provide some insight. 

https://stevenloftus.com/projects/hotel-hacking-cvent-passkey/

**Comment by overallcoma** [comment]
My bad - that was my report.

I've forked the project and will be offering a Python3 based alternative that facilitates the same functionality based off the code I wrote that led to this report.

Edit - fork is set up.  It isn't really a fork, kind of a full re-write.  It uses Python 3, I just stole part of the class out of this script and wrote the rest.  So it isn't a clone, the features and setup are VERY different.
